Client/Frontend/Browser/UserScriptManager.swift

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import WebKit
+
+class UserScriptManager {
+    init(tab: Tab) {
+        // All Frames (.atDocumentStart)
+        if let path = Bundle.main.path(forResource: "AllFramesAtDocumentStart", ofType: "js"),
+            let source = try? NSString(contentsOfFile: path, encoding: String.Encoding.utf8.rawValue) as String {
+            let userScript = WKUserScript(source: source, injectionTime: .atDocumentStart, forMainFrameOnly: false)
+            tab.webView?.configuration.userContentController.addUserScript(userScript)
+        }
```

Comment:
I was just about to do this change, but indicating whether or not the script is for the main frame only is not defined as part of the `WKUserScriptInjectionTime` enum. In that case, it might be clearer to leave this as-is.